cypress/integration/2-page-component-tests/checkboxes.cypress.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path')
+
+const { waitForApplication } = require('../utils')
+
+const templatesView = path.join(Cypress.config('fixturesFolder'), 'views', 'checkbox-test.html')
+const appView = path.join(Cypress.env('projectFolder'), 'app', 'views', 'checkbox-test.html')
+
+const pagePath = '/checkbox-test'
+
+describe('checkbox tests', () => {
+  before(() => {
+    waitForApplication()
+
+    // load test view
+    cy.task('deleteFile', { filename: appView })
+    cy.task('log', `Copy ${templatesView} to ${appView}`)
+    cy.task('copyFile', { source: templatesView, target: appView })
+    cy.task('log', 'The checkbox-test page should be displayed')
+    cy.visit(pagePath)
+    cy.get('h1')
+      .should('contains.text', 'Checkbox tests')
+  })
+
+  after(() => {
+    waitForApplication()
+    cy.task('deleteFile', { filename: appView })
+  })
+
+  const submitAndCheck = async (matchData) => {
+    cy.intercept('POST', pagePath).as('submitPage')
+    cy.get('button[data-module="govuk-button"]')
+      .should('contains.text', 'Continue')
+      .click()
+    cy.wait('@submitPage').its('request.body')
```

Comment:
This is cool. The request still makes it to the server right?

Reply:
I think it does